site/source/partials/example-code-header.hbs

```diff
@@ -6,7 +6,7 @@
 
   <!-- Load D3 and vega -->
   <script type="text/javascript" src="http://cdnjs.cloudflare.com/ajax/libs/d3/3.5.5/d3.min.js"></script>
-  <script type="text/javascript" src="http://cdnjs.cloudflare.com/ajax/libs/vega/1.4.3/vega.min.js"></script>
+  <script type="text/javascript" src="http://cdnjs.cloudflare.com/ajax/libs/vega/2.5.2/vega.min.js"></script>
```

Comment:
This version is inconsistent w/ the [version of vega declared in package.json - ~2.0.1](https://github.com/Esri/cedar/blob/develop/package.json#L63). Also, so is the version of d3 on the line above.